.github/workflows/push.yml

```diff
@@ -0,0 +1,72 @@
+name: Push
```

Comment:
Let me know if you prefer tabs here.

Reply:
I have no idea what are the "tabs" :)

Reply:
Instead of spaces I mean 😊 You seem to be using tabs almost exclusively for indentation.

Reply:
ahh :) I don't care. spaces are good.

I use the PHP internals indentation rules.